tests/utils/unittest_ast_walker.py

```diff
@@ -7,7 +7,7 @@
 
 import astroid
 
-from pylint.checkers.utils import check_messages
+from pylint.checkers.utils import only_required_for_messages
```

Comment:
Shall we add one test with `pytest.warns(DeprecationWarning)`? That means that coverage will remain 100% for the `check_messages` function and tests the deprecation.

Reply:
Yes, that makes sense - will do!